benchmark_stream.go

```diff
@@ -37,11 +39,12 @@ type benchmarkStreamMethod struct {
 
 // Call dispatches stream request on the provided transport.
 func (m benchmarkStreamMethod) Call(t transport.Transport) (benchmarkCallResult, error) {
-	streamIO := newStreamIOBenchmark(m.streamRequestMessages)
+	callResult := &streamCallResult{}
+	streamIO := newStreamIOBenchmark(m.streamRequestMessages, callResult)
```

Comment:
Is there another solution than giving callResult to `newStreamIOBenchmark`? The way callResult is updated is not consistent, one part is done in this method `Call` (with the latency) and the other part in the struct `StreamIOBenchmark`.

Reply:
good catch, I fixed this by adding additional methods in streamIO to get send and received message count and callResult is created in `Call` method. This seems much more cleaner.